### ISSUE TITLE
Annotation File of Thumos - background class and ignore class

### ISSUE THREAD

**Issue by HYUNJS** [TuanTNG/SimOn#2]
Thank you for sharing the codebase for your wonderful work! 

While reproducing SimOn, I found that the background class is sometimes labeled as False even though there is no action in your shared multiclass annotation files. Regarding that I have the following two questions:
1. How did you generate multiclass annotation files to label the background and ignore classes as True?
2. Is there any reason you do not train a classifier for the background class?(https://github.com/TuanTNG/SimOn/blob/main/train.py#L35)

Thank you :)

**Comment by TuanTNG** [comment]
Hi, 

Thank you for your interest in our work.

I answer your question as follows.
1. I use the official thumos.json file from [ActionFormer](https://github.com/happyharrycn/actionformer_release), read the ground-truth segments then generate the multiclass annotation files. The background class in the file is not important because I don't use it in the code, thus it may incorrect. Sorry for the confusion. 
2. I use Binary Cross Entropy loss, so the background class should not be trained. Let's say, there are C classes, the predicted vector will have the length of C. If all the values in the predicted vector are 0, then it will be the background class.

If you have any further questions, please let me know.
